### ISSUE TITLE
Stranglethorn Fishing Extravaganza

### ISSUE THREAD

**Issue by pilnahn** [WoWManiaUK/Redemption#2565]
The Fishing Extravaganza started at 14:00 today, as the npc Riggle Bassbait yelled out the starting of the competition. Where on previous attempts, many School of Tastyfish spawned from the beginning, there were none, along the entire coast of stranglethorn (I checked the entire west coast, which is where the majority spawn). A winner was not announced, and 15 mins after the start of the event I returned to Booty Bay to check on the npc, and he was saying that there was already a winner. As there were no schools, this is impossible.

**Comment by Keader** [comment]
So... was not School of Tastyfish?
I cant reproduce.
Forcing event start i can see Schools in correct places

![image](https://user-images.githubusercontent.com/7120913/42174628-65c7d1ae-7df9-11e8-875a-877a09041722.png)

**Comment by squalllh1989** [comment]
no, there is school of tasty fish, i run across many of them, just when people can't find them they start complain that "there is none" >.<

**Comment by squalllh1989** [comment]
This is the same happen with kaluak fishing derby, when some one keep fishing for the shark but he can't get one, after a while he will start to doubt that "is it really working ?" but it actually work, i personally got the shark and win the event once, and run across many school of tasty fish at stranghornvale, so i know for sure that they are there. But the only thing i need to double check is to see when they spawn, right after the time start or 1 hour later or something, will gonna check that next event.

**Comment by pilnahn** [comment]
Bruh, I went up and down the coast, there were no schools. I checked every school. this bs about 'not being able to find them' is just that; bs. if I went all the way up and down, couldn't find a single one, nor could the person looking with me, and there were no other people in stv bar the ones levelling (who wouldn't have made it all the way up and down fishing them in the pools that _didn't exist_ on a ground mount then making it back to booty bay in 15 mins), yes I checked horde too I'm not an idiot, I mean come on, are you seriously saying that it works? If so then you're deluded quite frankly.
